generators/entity-server/index.js

```diff
@@ -71,34 +73,81 @@ module.exports = class extends BaseBlueprintGenerator {
         return this._writing();
     }
 
+    _preparing() {
+        return {
+            processJavaEntityImports() {
+                this.importJsonIgnore = false;
+                this.importJsonIgnoreProperties = false;
+                this.importSet = false;
+                this.uniqueEnums = {};
+
+                this.importApiModelProperty = Object.values(this.relationships).filter(v => typeof v.javadoc != 'undefined').length > 0;
+                if (!this.importApiModelProperty) {
+                    this.importApiModelProperty = Object.values(this.fields).filter(v => typeof v.javadoc != 'undefined').length > 0;
+                }
+
+                Object.values(this.relationships).forEach(v => {
+                    if (v.ownerSide === false && ['one-to-many', 'one-to-one', 'many-to-many'].includes(v.relationshipType)) {
+                        this.importJsonIgnore = true;
+                    } else if (v.relationshipType === 'many-to-one') {
+                        this.importJsonIgnoreProperties = true;
+                    }
+                    if (v.relationshipType === 'one-to-many' || v.relationshipType === 'many-to-many') {
+                        this.importSet = true;
+                    }
+                });
+
+                Object.values(this.fields).forEach(v => {
```

Comment:
Rename `v` to `relationship`.
```suggestion
               this.fields.forEach(v => {
```